docs/release_notes/9_2.md

```diff
@@ -41,9 +82,7 @@ Some highlights and new features from this release are detailed next.
 ### Installer and Image Creation
 
 * Image builder now offers a new and improved way to create blueprints and images in the image builder web console.
-* Creating customized files and directories in the /etc directory is now supported.
-
-Some highlights and new features from this release are detailed next.
+* Creating customized files and directories in the `/etc` directory is now supported.
```

Comment:
at line 119 below the spelling for changes should be changed
Section Compilers and ...:
For details on the **changes** in this category see the